src/test/java/Homework19.java

```diff
@@ -0,0 +1,44 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest{
+    @Test
+    public void deletePlaylist() throws InterruptedException{
+        String expectedPlaylistDeletedMessage = "Deleting playlist \"TestPro Playlist\"";
```

Comment:
Hello,
Your homework 19 has been reviewed, but the test script failed. Please recheck and correct it.
![image](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/144403954/0921e1c8-b507-4c2c-b932-37065313aa37)